WordPressAuthenticator/Analytics/AuthenticatorAnalyticsTracker.swift

```diff
@@ -234,34 +234,9 @@ public class AuthenticatorAnalyticsTracker {
     /// Shared Instance.
     ///
     public static var shared: AuthenticatorAnalyticsTracker = {
-        return AuthenticatorAnalyticsTracker(configuration: defaultConfiguration())
```

Comment:
I'm simplifying the configuration of the tracker since we no longer have multiple feature flags to account for.